lib/temporal/connection.rb

```diff
@@ -0,0 +1,287 @@
+require 'temporal/api/workflowservice/v1/request_response_pb'
+require 'temporal/bridge'
+
+module Temporal
+  class Connection
```

Comment:
When you get around to unit tests, I have a suggested one for you (modeled off of https://github.com/temporalio/sdk-python/blob/431ca1967d365556a9cf5aa9aac00243b71059f8/tests/test_workflow_service.py#L19 but maybe even better). Create a Ruby gRPC server from workflow service, call all of your RPC calls directed to it, and confirm all were called accurately. May also have to add a client interceptor to confirm the request type and response type.

This can help you feel good about: 1) knowing you didn't fat finger any RPC names, 2) you got the mapping between request and response right, and 3) you aren't missing any new gRPC calls.

Reply:
Yes, this is exactly how I was planning to test it, thank you for the reference 🙌